app/models/spree_avatax_certified/address.rb

```diff
@@ -137,5 +137,12 @@ def resolve_stock_location
       package = Spree::Stock::Coordinator.new(@order).packages.find { |package| package.line_items.any? {|li| li.id == @line.id} }
       package.stock_location
     end
+
+    def resolve_item_ship_to_address
+      shipment = @order.shipments.select { |s| s.line_items.include?(@line) }
+      return shipment.address if shipment.present?
```

Comment:
How does the `Shipment#address` relationship works? Does each shipment has an address that matches the destination? Eg: 
- walkout shipment with store address?
- pickup shipment with store address?

Reply:
AFAIK yes, for POS orders the shipment address is the store and for web orders the shipment address is the shipping address

Reply:
same for BOPIS i believe, will try to find an order for testing that case